src/app/action/action.component.html

```diff
@@ -24,7 +24,7 @@
   <button class="btn btn-link dropdown-toggle" type="button" dropdownToggle
           [ngClass]="{'disabled': config.moreActionsDisabled}"
           (click)="initMoreActionsDropup($event)"
-          aria-label="More Actions">
+          [attr.aria-label]="config.ariaLabel || 'More Actions'">
```

Comment:
I don't believe we should provide the "More Actions" string by default. Unless we commit to localizing our components, hard coded strings don't work well across applications. This would break the UI as soon as someone changes the preferred browser locale. We don't provide default button labels; for example, even though it would look silly without one.

That said, if we had a moreActionsTooltip property, perhaps that could be used as a default.